Godeps/_workspace/src/github.com/jackc/pgx/bench_test.go

```diff
@@ -1,7 +1,7 @@
 package pgx_test
 
 import (
-	"github.com/flynn/flynn/Godeps/_workspace/src/github.com/jackc/pgx"
+	"github.com/jackc/pgx"
```

Comment:
Looks like some paths got rewritten backwards, please drop any changes like this from the patch.